### PR TITLE
Translate CDN Links page

### PR DESCRIPTION
**Pull request by pachabhaiya**
Translating the [CDN Links](https://ne.reactjs.org/docs/cdn-links.html) page in Nepali language.
Please do provide feedback.

### CODE REVIEW

**Comment by samundrak** [comment]
this is awesome, Thanks

File: content/docs/cdn-links.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 ```
 
-To load a specific version of `react` and `react-dom`, replace `16` with the version number.
+`react` र `react-dom` को विशेष संस्करण प्रयोग गर्नको लागि, `16` लाई संस्करण अंकले बदल्नुहोस।
 
-### Why the `crossorigin` Attribute? {#why-the-crossorigin-attribute}
```
Comment (nootanghimire) [inline content/docs/cdn-links.md]: I don't know if we should translate `Attribute` because it has some meaning attached to it?

Comment (pachabhaiya) [reply content/docs/cdn-links.md]: Agreed. Keeping `Attribute` as it is makes more sense to me too.
I'll update the PR.

**Comment by pachabhaiya** [comment]
Thanks @samundrak and @nootanghimire for reviewing this PR.
Since it is approved, I'll go ahead and merge this in.

Please let me know if there is any other process that needs to be followed for merging the PR.

**Pull request merged**